packages/schematics/webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const path = require('path');
+const ProgressPlugin = require('webpack/lib/ProgressPlugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+// Environment config
+const NODE_ENV = process.env.NODE_ENV || 'production';
+const isProduction = NODE_ENV === 'production';
+
+console.log(`
+  Building Augury Labs Schematics in ${NODE_ENV} mode
+`);
+
+/*
+ * Config
+ */
+module.exports = {
+  mode: isProduction ? 'production' : 'development',
+  devtool: isProduction ? false : ' source-map',
+  target: 'node',
+  cache: true,
+  context: __dirname,
+  stats: {
+    colors: true,
+    reasons: true,
+  },
+
+  externals: {
+    'aws-sdk': 'aws-sdk',
```

Comment:
Can we add a comment here describing this is necessary to workaround an issue with building.

Reply:
Yeah, will do.